standard/sections/clause_6_informative_text.adoc

```diff
@@ -2,45 +2,35 @@
+* *(1) WIS 2.0 adopts Web technology and leverages industry best practices and open standards.*
+* *(2) WIS 2.0 uses Uniform Resource Locators (URL) to identify resources (i.e., Web pages, data, metadata, APIs) use.*
+* *(3) WIS 2.0 prioritizes the use of public telecommunications networks (i.e. Internet) when publishing digital resources.*
+* *(4) WIS 2.0 requires provision of Web service(s) to access or interact with digital resources (e.g. data, information, products) published using WIS.*
+* (5) WIS 2.0 encourages NCs and DCPCs to provide 'data reduction' services via WIS that process 'big data' to create results or products that are small enough to be conveniently downloaded and used by those with minimal technical infrastructure.
+* *(6) WIS 2.0 adds open standard messaging protocols that use the publish-subscribe message pattern to the list of data exchange mechanisms approved for use within WIS and GTS.*
+* (7) WIS 2.0 requires all services that provide real-time distribution of messages (containing data or notifications about data availability) to cache/store the messages for a minimum of 24 hours and allow users to request cached messages for download.
+* (8) WIS 2.0 adopts direct data-exchange between provider and consumer and phases out the use of routing tables and bulletin headers.
+* *(9) WIS 2.0 provides a catalogue containing metadata that describes both data and the service(s) provided to access that data.*
+* *(10) WIS 2.0 encourages data providers to publish metadata describing their data and Web services in a way that can be indexed by commercial search engines.*
 
 ==== Discovery metadata design considerations
 
 The following describes envisioned workflows of WIS 2.0 in the context of metadata search and harvesting.
 
-We consider the following:
-
-* flexible metadata publishing mechanisms: providers need to be able to publish discovery metadata in the easiest and
-most efficient way possible
-* basic, HTTP crawlable metadata files (filesystem, object storage). For example, publishing discovery metadata as
-JSON files to an S3 bucket, and then making that bucket available for harvesting and traversal to search engines and
-metadata harvesters
-* the browser as the catalogue: here, browsers utilize mass market search engines as the gateway to low barrier
-discovery.  This pattern may work by way of the Global Discovery Catalogue and the Global Cache, with the idea that
-search engines can index from the authoritative source
+* Flexible metadata publishing mechanisms so providers can publish discovery metadata in the easiest and most efficient way possible.
+* Basic, HTTP crawlable metadata files (filesystem, object storage). For example, publishing discovery metadata as
+JSON files to an S3 bucket, and then making that bucket available for harvesting and traversal to search engines and metadata harvesters.
```

Comment:
Remove trailing periods from this bulleted list

Reply:
I'm leaving the periods in the lists.